packages/dialtone-vue2/recipes/conversation_view/emoji_row/emoji_row.vue

```diff
@@ -30,7 +40,7 @@
           >
             <span class="d-recipe-emoji-row__emoji">
```

Comment:
hmm I think we will need to increase the vertical padding / height now that we made the emoji bigger. It looks a bit too tight to the edge. Consult with design if needed.